[DRAFT] New Learn: Cascade layers

estelle opened the pull request:
<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->

#### Related issues
<!-- 🔨 If applicable, use "Fixes #XYZ" -->

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

MrBrain295 commented on files/en-us/learn/css/building_blocks/cascade_layers/index.md:
```diff
@@ -0,0 +1,235 @@
+---
+{{LearnSidebar}}{{NextMenu("Learn/CSS/Building_blocks/Selectors", "Learn/CSS/Building_blocks")}}
+
+The aim of this lesson is to develop your understanding of some of the most fundamental concepts of CSS — the cascade, specificity, and inheritance — which control how CSS is applied to HTML and how conflicts between style declarations are resolved.
```
```suggestion
The aim of this lesson is to develop your understanding of some of the most fundamental concepts of CSS—the cascade, specificity, and inheritance—which control how CSS is applied to HTML and how conflicts between style declarations are resolved.
```

teoli2003 replied:
The original is better.

MrBrain295 replied:
https://www.uis.edu/learning-hub/writing-resources/handouts/learning-hub/dashes-and-parentheses

MrBrain295 commented on files/en-us/learn/css/building_blocks/cascade_layers/index.md:
```diff
@@ -0,0 +1,235 @@
+ 
+## Quick recap: the cascade 
+ 
+The cascade, the C in CSS, is the method by which styles cascade together. The user agent goes through several very clearly defined steps to determine which single value gets assigned for every property of every element. We will briefly list the steps, then dig deeper into step 4, cascade layers, which is what you came here to learn:
```
```suggestion
The cascade, the C in CSS, is the method by which styles cascade together. The user agent goes through several very clearly defined steps to determine which single value gets assigned to every property of every element. We will briefly list the steps, then dig deeper into step 4, cascade layers, which is what you came here to learn:
```

estelle commented:
Closes https://github.com/openwebdocs/project/issues/94

rubiesonthesky committed:
fix: Add some missing parentheses for arrow functions (#20278)

paulmillr committed:
Bigints: update cryptography usage. (#20272)

* Bigints: update cryptography usage.

See #18494

* Update files/en-us/web/javascript/reference/global_objects/bigint/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Josh-Cena committed:
Add section on string coercion behavior (#20234)

* Add section on string coercion behavior

* Copy edits

* Remove mention of ToString

* Deduplicate

weslord committed:
Remove "unloaded" from visibilityState values on Page Visibility API (#20284)

teoli2003 committed:
Remove prerender value (#20285)

Josh-Cena committed:
Add a section on checking is regex (#20281)

* Add a section on checking is regex

* Fixes

Josh-Cena committed:
Elaborate on how RegExp.prototype.source is escaped (#20282)

Josh-Cena committed:
Reorganize arguments docs (#20277)

GeorgeDaris committed:
Added information + Typo fixes (#20173)

* Added information + Typo fixes

* Update files/en-us/learn/accessibility/wai-aria_basics/index.md

Co-authored-by: Scott O'Hara <scottaohara@users.noreply.github.com>

* Update index.md

Corrected the wording for the browser support

* Update files/en-us/learn/accessibility/wai-aria_basics/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/learn/accessibility/wai-aria_basics/index.md

* Update files/en-us/learn/accessibility/wai-aria_basics/index.md

Co-authored-by: Scott O'Hara <scottaohara@users.noreply.github.com>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>
Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Josh-Cena committed:
Mention Weak data structures in memory management (#20147)

* Mention Weak data structures in memory management

* Mention web pages

* Elaborate on theoretical stuff

* Change heading text

* Update files/en-us/web/javascript/memory_management/index.md

* Update files/en-us/web/javascript/memory_management/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

ras-2004-code committed:
Fix comment (#20287)

On line 588, in the last example under the Functions heading, just above the Anonymous Functions heading, the value for avg(1,2) is given as 1.5 in the comments.
But considering the function defined above the sum is divided by 3 regardless of number of arguments. So it will return 1.

de-oz committed:
Fix punctuation (#20290)

de-oz committed:
Remove extra spaces (#20289)

Tolhos committed:
Remove deprecated methods from managing_screen_orientation (#20295)

Co-authored-by: Michael[tm] Smith <mike@w3.org>

Josh-Cena committed:
Correct description about function displayName (#20297)

hinell committed:
add: WebAPI/URL Links to encodeURI (#20294)

Co-authored-by: de-oz <101826623+de-oz@users.noreply.github.com>

teoli2003 committed:
Simplify Page Visibility API (#20286)

OnkarRuikar committed:
Provide sidebars to WebDriver pages (#20271)

Commit:
Highlight note re: default vh unit (#20269)

Josh-Cena committed:
Refer to all instance methods with .prototype. (#20292)

OnkarRuikar committed:
Fix typos (#20300)

* Fix typos

* use 'some of the'

Josh-Cena committed:
Improve jsxref macro typesetting (#20298)

nschonni committed:
fix: MD001 Headings should only increment by 1 (#20301)

nschonni committed:
fix: MD024 Multiple headings with same content (#20118)

* fix: MD024 Multiple headings with same content

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/css/css_container_queries/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix: Re-add HTML/JavaScript headings

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

OnkarRuikar committed:
Add custom replace rule to markdownlint configuration (#17988)

* Add replace rule to markdownlint task

* Update yarn.lock, commands, and add more rules

* Add the dependency

* bump dependancy version

* deleberate error for testing

* revert the test commit

OnkarRuikar committed:
Canvas: update BCD info (#20302)

nschonni committed:
fix: MD049 consistent emphasis style (#20105)

* fix: MD049 consistent emphasis style

* fix: MD049 consistent emphasis style

* Update files/en-us/learn/server-side/django/forms/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/accessibility/mobile_accessibility_checklist/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Apply suggestions from code review

* Update files/en-us/learn/server-side/django/forms/index.md

* Format MathML

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

sama7 committed:
Fixing typos (#20304)

nschonni committed:
fix: qualify bad internal links (#20306)

OnkarRuikar committed:
Markdownlint: Use the custom search-replace plugin in PR check (#20307)

OnkarRuikar committed:
Add missing features to the 'document' page. (#20305)

* Add missing features to the 'document' page.

* Update experimental badges

* Fix typos

* Fix a typo

* Lint errors to test the new lint plugin

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Fix the lint error

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

bsmth committed:
Update PULL_REQUEST_TEMPLATE (#20249)

* Update PULL_REQUEST_TEMPLATE

This PR removes the checklist items which are rendered as tasks in new pull requests

* Add reviewer feedback

Adding more details for for related issues and pull requests

* Clarify closing issue criteria

HolgerJeromin committed:
CSS.escape() has a return value (#20309)

* CSS.escape() has a return value

* Update files/en-us/web/api/css/escape/index.md

* Update files/en-us/web/api/css/escape/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

de-oz committed:
Fix typos (#20311)

fred-wang committed:
Add MathML tutorial (#19467)

* Add MathML tutorial

An introduction to basic MathML constructions, using similar structure
as the lessons for HTML, CSS and JavaScript. This only introduces the
"first steps" module but more advanced topics may be added in the
future.

* fix typo with sub/supp

* find -type f -name *.md | xargs sed -i 's/<html>/<html lang="en-US">/g'

* for x in $(find files/en-us/learn/mathml/first_steps -name *.png); do yarn filecheck "$x" --save-compression; done

* format files

* Don't introduce the display/inline term which are specific to LaTeX / math typesetting, but not mentioned in MathML and confusing with MathML display attribute CSS block/inline. Intead just say whether the <math> has a display="block" attribute or not. Also add an initial note about CSS math-style.

* Update files/en-us/learn/mathml/first_steps/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/getting_started/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/getting_started/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/tables/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/text_containers/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/fractions_and_roots/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/fractions_and_roots/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Update files/en-us/learn/mathml/first_steps/scripts/index.md

Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

* Address review comments from @Rumyra

* address review comments from brian

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>
Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

de-oz committed:
Fix spacing around periods (#20313)

mindplay-dk committed:
Encourage use of AbortController (#20315)

* update old text

with IE being officially dead, all major browsers support this API since 2019

* Update files/en-us/web/api/fetch_api/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

dipikabh committed:
Fix typos (#20319)

The incorrect example should not have the comma before "which"

Josh-Cena committed:
Consistently use 3 dots for for...of/in (#20317)

Josh-Cena committed:
Correct description of default parameters scope (#20318)

* Correct description of default parameters scope

* Update files/en-us/web/javascript/reference/functions/default_parameters/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Josh-Cena committed:
Fix typos (#20320)

Rumyra committed:
Updating PAB (#20310)

* updating pab

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mdn/mdn_product_advisory_board/index.md

* Format

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
Co-authored-by: Brian Thomas Smith <brian@smith.berlin>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

dependabot[bot] committed:
Bump tj-actions/changed-files from 29.0.2 to 29.0.3 (#20322)

dependabot[bot] committed:
Bump @mdn/yari from 1.18.9 to 1.18.14 (#20323)

rubiesonthesky committed:
fix: missing ");" from code (#20325)

nschonni committed:
fix: remove OL for code fences (#20324)

bsmth committed:
Clarify Z-axis in context of CSS coordinates (#20321)

* chore: 3367 - clarify z-axis for CSS positioning

* Update files/en-us/web/css/cssom_view/coordinate_systems/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
fix: various typos (#20327)

OnkarRuikar committed:
Fix typos (#20335)

* Fix typos

* Update files/en-us/games/techniques/control_mechanisms/desktop_with_gamepad/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Josh-Cena committed:
Consistently use en.wikipedia.org domain (#20332)

sama7 committed:
Fixing Typos (#20336)

Official spelling of the full form of CSS is "Cascading Style Sheets"
https://www.w3.org/Style/CSS/

sama7 committed:
Fixing typo (#20337)

Official spelling of the full form of CSS is "Cascading Style Sheets"
https://www.w3.org/Style/CSS/

Josh-Cena committed:
Consistently explain the domain and range of trig-related functions (#20331)

* Consistently explain the domain and range of trig-related functions

* Fixes

* Fixes

Josh-Cena committed:
Consistently explain the domain and range of power-related functions (#20338)

* Consistently explain the domain and range of power-related functions

* Update files/en-us/web/javascript/reference/global_objects/math/log1p/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

sama7 committed:
Fixing typos (#20339)

sama7 committed:
Fixing typo (#20340)

Josh-Cena committed:
Unify style of remaining Math pages (#20341)

ramiy committed:
`tan()` - New CSS functional notation (#19145)

* `tan()` - New CSS functional notation

* Update index.md

* Update index.md

* Add "Argument" section

* Update index.md

* Add live example

* live example height

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

Natnaelk committed:
fixed a typo on find out more section (#20344)

sideshowbarker committed:
Add an overview page for the Paint Timing API (#20346)

* Add an overview page for the Paint Timing API

Fixes https://github.com/mdn/content/issues/10287

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

fred-wang committed:
Link MathML tutorial from the MathML and Learn pages. (#20349)

nschonni committed:
fix: MD036 emphasis used instead of a heading (#20099)

* fix: MD036 emphasis used instead of a heading

* fix: delete old content

Co-authored-by: wbamberg <will@bootbonnet.ca>

* fix: remove BR in heading

Co-authored-by: wbamberg <will@bootbonnet.ca>

* fix: restore emphasis with punctuation

* fix: MD012 from deletions

* fix: listify manifest examples

* fix: formatting from code review

* Update files/en-us/mdn/community/issues/index.md

* Update index.md

Co-authored-by: wbamberg <will@bootbonnet.ca>
Co-authored-by: Ruth John <Rumyra@users.noreply.github.com>

ytxmobile98 committed:
docs: Web Worker APIs doc in bullet list format (#20345)

* docs: use bullet points for supported web APIs

Use bullet points to list supported APIs, for better layout and readability

* docs: functions available to all workers

* fix: use - as bullet points

* fix: performance APIs layout

* fix: spaces

{{domxref("URL_API","URL API")}} (e.g. {{domxref("URL")}})

* Update files/en-us/web/api/web_workers_api/index.md

* Make Markdownlint happy

* Update files/en-us/web/api/web_workers_api/index.md

* Fix bullets

`TextEncoder` and `TextDecoder` are examples of `Encoding_API`

* Typo on TextEncoder xref

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

queengooborg committed:
Remove api.CanvasRenderingContext2D.drawWindow from MDN (#20352)

noamr committed:
Add more reasons for first contentful paint (#20351)

noamr committed:
Make first-paint glossary more precise (#20350)

First paint is measured from the first time something other than the body background color is painted, not from the first time something other than the previous page is painted.

de-oz committed:
Remove extra spaces (#20363)

de-oz committed:
Remove punctuation marks from code (#20362)

Josh-Cena committed:
Deemphasize IE notes in typeof, more descriptive headings (#20364)

knenkne committed:
[Role: Grid] More information about keyboard interactions (#20141)

* [Role: Grid] update keyboard interactions table & calendar demo

* [Role: Grid] removing formating changes

dependabot[bot] committed:
Bump @mdn/yari from 1.18.14 to 1.18.19 (#20353)

* Bump @mdn/yari from 1.18.14 to 1.18.19

Bumps [@mdn/yari](https://github.com/mdn/yari) from 1.18.14 to 1.18.19.
- [Release notes](https://github.com/mdn/yari/releases)
- [Changelog](https://github.com/mdn/yari/blob/main/docs/npm-releases.md)
- [Commits](https://github.com/mdn/yari/compare/v1.18.14...v1.18.19)

---
updated-dependencies:
- dependency-name: "@mdn/yari"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* fix formatting

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

spectranaut committed:
Suggestion is not a landmark role (#20224)

See the specification: https://w3c.github.io/aria/#suggestion

wbamberg committed:
Fix 20365: remove mention of Flash (#20367)

OnkarRuikar committed:
Daily markdownlint (#20369)

wbamberg committed:
Fix some GitHub Pages links (#20373)

wbamberg committed:
Remove some more Flash references (#20372)

Josh-Cena committed:
Mention return type of derived class constructor and non-object prototype (#20371)

PositiveUser committed:
Fix typo in project writing guidelines (#20366)

dawei-wang committed:
Added ElementInternals.role to section (#20182)

OnkarRuikar committed:
Synchronize with BCD v5.2.1 (#20370)

nschonni committed:
chore: migrate to markdownlint-cli2 (#20330)

* chore: migrate to markdownlint-cli2

* typo: algorythim -> algorithm

Co-authored-by: Gabe <66077254+MrBrain295@users.noreply.github.com>

* chore: update disable reasons

Co-authored-by: Claas Augner <495429+caugner@users.noreply.github.com>

* chore: link JSONC in addition to JSON

Co-authored-by: Gabe <66077254+MrBrain295@users.noreply.github.com>
Co-authored-by: Claas Augner <495429+caugner@users.noreply.github.com>
Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

dawei-wang committed:
Fixed word usage (#20357)

Changed HTTP pipelining to protocol as suggested 

Closes mdn/content#20326

bsmth committed:
4069: add note about HTML namespaces, point to namespace reference (#20355)

nschonni committed:
chore: Add Markdownlint cleanup job (#16036)

* chore: Add Markdownlint cleanup job

yarn cache

chore: use mdn-bot as author

chore: update PR titles and branch

* chore: add direct link to failing job

Co-authored-by: Claas Augner <495429+caugner@users.noreply.github.com>

Co-authored-by: Claas Augner <495429+caugner@users.noreply.github.com>

MatiasWengiel committed:
Update to include links to `type` and `herflang` pages (#20356)

nschonni committed:
ci(workflows): merge Preview into Check scripts workflow (#20328)

* chore: remove preview.yml

* ci: migrate preview calls to pr-check_scripts

fred-wang committed:
Add property math-shift (#20314)

* Add property math-shift

https://w3c.github.io/mathml-core/#the-math-shift

* Update index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

estelle committed:
Minor grammatical tweaks for accuracy (#20375)

estelle committed:
Missing alt attributes (#20376)

* Missing alt attributes

alt text for six images
removed mention of IE.

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Josh-Cena committed:
Fix trailing comma description about quantifier, add description (#20377)

* Fix trailing comma description about quantifier, add description

* Update files/en-us/web/javascript/reference/trailing_commas/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Jkat committed:
Fix typo in host_permissions (#20379)

de-oz committed:
Improve spelling (#20387)

sideshowbarker committed:
Add API overview page for Device Orientation Events (#20385)

* Add API overview page for Device Orientation Events

Fixes https://github.com/mdn/content/issues/2644

* Update all links to the existing guide pages we moved

nschonni committed:
chore: remove orphan .eslintrc.js (#20381)

jasonren0403 committed:
Fix a link error (#20389)

fred-wang committed:
Fix legacy algorithm for <semantics> and remove obsolete attributes (#20347)

* Fix legacy algorithm for <semantics> and remove obsolete attributes

Update the note about the legacy algorithm implemented in WebKit and
Firefox, which actually skip the annotations that have an `src`
attribute [1] [2]. Also try to clarify that <annotation-xml> without
a known encoding are skipped too, as current text is not clear.

MathML Core only mentions `encoding` for non-global attributes
accepted on annotation/annotation-xml elements. So mark `src` as
deprecated and remove all the other attributes that were never
involved in browser implementations.

[1] https://github.com/WebKit/WebKit/blob/b7d555805988f56bc97e9ae21014a35a463b1987/Source/WebCore/mathml/MathMLSelectElement.cpp#L174
[2] https://searchfox.org/mozilla-central/rev/3f9dcc016dd96a0336d46f4a19aeabdd796ab9e9/layout/mathml/nsMathMLsemanticsFrame.cpp#63
[3] https://w3c.github.io/mathml-core/#semantics-and-presentation

* Update files/en-us/web/mathml/element/semantics/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

de-oz committed:
Remove extra spaces (#20390)

ramiy committed:
Update the parameters of CSS `sin()` function (#20386)

* Update the parameters of CSS `sin()` function

* Run Prettier

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

ramiy committed:
Update trigonometric functions description (#20388)

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

yin1999 committed:
remove the broken link (#20391)

ra-jeev committed:
Fixing typo in Operators explanation (#20392)

The result on line no. 254 as shown in the comment is wrong. The correct value should be 0

Original:
```js
const a = 0 && "Hello"; // "Hello" because 0 is "falsy"
```

Proposed:
```js
const a = 0 && "Hello"; // 0 because 0 is "falsy"
```

dependabot[bot] committed:
Bump @mdn/yari from 1.18.19 to 1.18.23 (#20393)

mganss committed:
Fix description of PNG image format (#20394)

PNG is lossless.

Josh-Cena committed:
Move shells page into technologies overview, remove outdated links (#20395)

bsmth committed:
1637: preventDefault on `keypress` example allowed browser autocomplete before any keystrokes (#20384)

* 1637 fix preventDefault example by disabling autocomplete

* 1637 add a sentence describing why we're disabling autocomplete

* Update files/en-us/web/api/event/preventdefault/index.md

Co-authored-by: Joe Medley <jmedley@google.com>

* Cleanup following reviewer feedback

Co-authored-by: Joe Medley <jmedley@google.com>

rebloor committed:
Private browsing window access permission clarifications (#19980)

yashikabhargava committed:
fix grammatical issues in index.md (#20399)

jccr committed:
Fix devicechange event sample (#20412)

Josh-Cena committed:
Elaborate on Function.p.bind behavior (#20400)

Josh-Cena committed:
Expand on in operator for branded checks (#20402)

OnkarRuikar committed:
fix syntax error (#20413)

de-oz committed:
Remove extra spaces (#20410)

sama7 committed:
Fixing typo (#20416)

sama7 committed:
Fixing typo (#20415)

sama7 committed:
Fixing typos (#20414)

* Fixing typos

* Update files/en-us/learn/html/introduction_to_html/getting_started/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

OnkarRuikar committed:
CSS: update Hash value definition syntax (#20417)

ramiy committed:
`asin()` - New CSS functional notation (#20383)

* `asin()` - New CSS functional notation

* remove example section

* Update files/en-us/web/css/asin/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/web/css/asin/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/web/css/asin/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/web/css/asin/index.md

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Example added

* Update index.md

* Update index.md

* Apply suggestions from code review

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

Josh-Cena committed:
Mention BigInts and symbols in Object.is() (#20401)

bsmth committed:
19707 remove 'obsolete' admonition, add contextmenu to 103 relnote (#20421)

ramiy committed:
`acos()` - New CSS functional notation (#20420)

* `acos()` - New CSS functional notation

* Update index.md

* Update index.md

fred-wang committed:
Update documentation for the merror element (#20426)

* Update documentation for the merror element

- Remove the confusing about "XML error message" and instead elaborate
  a bit on the description, as worded in the MathML spec.

- Tweak a bit the example, explaining it in English and with a
  a live preview.

* use the example from MathML Core

fred-wang committed:
Update documentation for `<mmultiscripts>` (#20427)

* Rewrite description of mmultiscripts, removing confusing description of tensors to focus on the layout.
* Replace regexp syntax with English and ellipsis/brace.
* Remove details on invalid markup.
* Remove doc for subscriptshift/superscriptshift attributes, this was overlooked in [2].
* Tweak a bit the examples, using live sample instead of PNG images and English explanation. Also add a more complex example so that shows the more general syntax and the order in which scripts are placed.

[1] https://w3c.github.io/mathml-core/#prescripts-and-tensor-indices-mmultiscripts
[2] https://github.com/mdn/content/pull/20161

sideshowbarker committed:
Add “controlslist” to <audio> element article (#20423)

Fixes https://github.com/mdn/content/issues/16768

sideshowbarker committed:
Add the “shadowroot” attribute to the <template> article (#20424)

Fixes https://github.com/mdn/content/issues/9559

Josh-Cena committed:
Elaborate on NaN behavior (#20429)

dependabot[bot] committed:
Bump tj-actions/changed-files from 29.0.3 to 29.0.4 (#20430)

dependabot[bot] committed:
Bump @mdn/yari from 1.18.23 to 1.18.28 (#20431)

fred-wang committed:
Update documentation for the `<mfrac>` element (#20425)

* Update documentation for the `<mfrac>` element

- MathML tutorial: fix use of token element in mfrac example
  (numbers should use mn element instead of mi element).

- Remove the `bevelled` attribute because it is considered
  irrelevant [1] [2].

- Mention that mfrac can be used for fraction-like objects
  too.

- Replace the example with the bevelled attribute with
  two examples for a simple mfrac and for the linethickness
  attribute. Use EmbedLiveSample instead of PNG image of
  a sample rendering.

[1] https://github.com/mdn/browser-compat-data/pull/17735
[2] https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#removal-of-irrelevant-features

* Update files/en-us/web/mathml/element/mfrac/index.md

Co-authored-by: Florian Scholz <fs@florianscholz.com>

* fixup! Update files/en-us/web/mathml/element/mfrac/index.md

Co-authored-by: Florian Scholz <fs@florianscholz.com>

ckashby committed:
Replace worked with 'working' (#20433)

use them in a ##worked## example
use them in a working example

rebloor committed:
Align managed storage snippet with favorite color example (#20254)

* Align managed storage snippet with favorite color example

* Addressed feedback

* Removed space

GeorgeDaris committed:
Fix typo (#20432)

P.S. There is no bold code in the "configuring our tools" section, at number 4. I followed along with the tutorial and the ESLint config file had the options listed automatically included, as well as the PostCSS one mentioned later on.

OnkarRuikar committed:
Fix a typo (#20446)

The character entity references has a typo.

OnkarRuikar committed:
Fix a code bug (#20448)

This prevents creation of implicit global variable `elem`.

nschonni committed:
style: metadata spacing for learn (#20451)

OnkarRuikar committed:
Update WebAssembly page (#20447)

* Update WebAssembly page

* Update files/en-us/webassembly/index.md

Co-authored-by: Hamish Willee <hamishwillee@gmail.com>

nschonni committed:
style: metadata spacing for mozilla (#20454)

nschonni committed:
style: metadata spacing for mdn (#20452)

nschonni committed:
style: metadata spacing for web/api/a-c (#20455)

nschonni committed:
style: metadata spacing for web/api/d-g (#20456)

OnkarRuikar committed:
avoid syntax errors (#20453)

nschonni committed:
style: metadata spacing for web/api/h-l (#20457)

nschonni committed:
style: metadata spacing for web/api/m-p (#20458)

nschonni committed:
style: metadata spacing for web/api/r-s (#20459)

nschonni committed:
style: metadata spacing for web/api/t-w (#20460)

nschonni committed:
style: metadata spacing for web/api/x (#20461)

nschonni committed:
style: metadata spacing for web/javascript (#20464)

OnkarRuikar committed:
Fix a bug in `pr-check_markdownlint` workflow (#20466)

nschonni committed:
style: metadata spacing for web/m-x (#20465)

nschonni committed:
style: metadata spacing for glossary (#20450)

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: metadata spacing for web/css (#20462)

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: metadata spacing for web/e-h (#20463)

mupple committed:
Replace references to `init` to conform with signature (#20469)

While the signature calls the second parameter `options`, the text called it `init`. This has been replaced with _options_ (for the parameter) and `myOptions` (for the argument, in line with the example code).

nschonni committed:
style: metadata spacing find/replace (#20468)

hamishwillee committed:
FF104 Add CSS Font Loading API in workers to Experimental features (#20471)

de-oz committed:
Remove extra spaces (#20474)

fred-wang committed:
Add redirects to mmultiscripts and semantics pages (#20472)

de-oz committed:
Fix typos (#20473)

fred-wang committed:
Document deprecate MathML 1 style attributes on <mstyle> (#20481)

* Document deprecate MathML 1 style attributes on <mstyle>

These elements are already listed on [1] and are still
implemented in WebKit. List them on <mstyle> too for completeness.

[1] https://developer.mozilla.org/en-US/docs/Web/MathML/Attribute

* address review comment

* fix typo

Elchi3 committed:
Rework HID event pages per new structure (#20478)

dependabot[bot] committed:
Bump @mdn/yari from 1.18.28 to 1.18.36 (#20484)

nschonni committed:
chore: Add VSCode suggested plugins (#20380)

collin-klippel committed:
fix: Change "code written" to "code executed" (#20487)

* fix: Change "in the browser" to "for the browser"

This copy highlights the versatility of JavaScript by contrasting
server-side and client-side applications. It refers to client-side
JavaScript as "written in the browser". It is more accurate to say
that client-side JavaScript is "written for the browser."

* fix: Add back correct text

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/web/javascript/guide/introduction/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

yashikabhargava committed:
fix grammatical issues in CSS layout learning (#20489)

* fix grammatical issues in index.md

* Update index.md

* Update files/en-us/learn/css/css_layout/introduction/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

cw118 committed:
Small typo fixes (#20490)

kaavyanannapaneni committed:
fixing a sentence (#20476)

* fixing a sentence

changed do not all implement new CSS at the same to ---
do not implement all of the new CSS features at the same time.

* Update files/en-us/learn/css/first_steps/how_css_works/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

OnkarRuikar committed:
fix a typo (#20449)

For correct values refer: https://drafts.csswg.org/css-counter-styles/#counter-style-speak-as

The specification link is pointing to the wrong fragment.

gampu committed:
Use present tense in "Applying CSS to the DOM" section (#20488)

sideshowbarker committed:
Make the kitchensink use browserc-compat, not Compat argument (#20480)

See https://github.com/mdn/yari/pull/6576#issuecomment-1241840985

AtofStryker committed:
fix: update XmlHttpRequest withCredentials documentation (#20486)

estelle committed:
Update page: alt text clarification and update to 2022 (#20406)

* Update page: alt text clarification and update to 2022

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* address comment

* Update files/en-us/learn/accessibility/html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

OnkarRuikar committed:
CSS: remove redundant properties and fix typos (#20498)

chaitanyaj2222 committed:
Added: alt attribute to few images (#20041)

* Added: alt attribute to few images

* Apply suggestions from code review

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>
Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

github-actions[bot] committed:
Update InterfaceData based on WebRef (#20497)

Co-authored-by: GitHub <noreply@github.com>

OnkarRuikar committed:
Fix a live sample (#20500)

* Fix a live sample

* Update the warning and use `<p>` for the demo

* add a lint error for testing the cron job

chriskirknielsen committed:
Capitalisation fix (#20495)

nschonni committed:
style: run prettier on Glossary (#20496)

* style: run prettier on Glossary

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

sideshowbarker committed:
Deprecate RTCIceServer»credentialType & RTCIceCredentialType (#20505)

https://github.com/w3c/webrtc-pc/commit/0cba163
(https://github.com/w3c/webrtc-pc/pull/2767) removed
RTCIceServer»credentialType and RTCIceCredentialType from the WebRTC spec.

Related BCD change: https://github.com/mdn/browser-compat-data/pull/17748

nschonni committed:
style: run prettier on webassembly (#20405)

nschonni committed:
typo: existant -> existent (#20507)

glmvc committed:
adapt "note box" due to a change in the interactive editor (#20491)

before a merged pr there was just one link with the target="_blank" attribute
now both of them have it

Josh-Cena committed:
Explain the control flow effect of await (#20503)

de-oz committed:
Fix grammar (#20511)

de-oz committed:
Fix wrong word order (#20510)

traelincoln committed:
Edit index.md to improve sentence clarity (#20509)

rubiesonthesky committed:
Add missing ); for forEach (#20518)

Ahmed-Hakeem committed:
fix deleting non-configurable properties section (#20439)

* fix deleting non-configurable properties section

let and const don't create any properties, delete operator return false because bindings defined with let and const has environment record [[Base]]

* Reorganize docs

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

YangFong committed:
Optimize the instance for array foreach() (#20513)

* Optimize the instance for array foreach()

* Move comment

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

rubiesonthesky committed:
Fix 2D collision detection example, can't use arrow function and this (#20517)

* Fix 2D collision detection example, can't use arrow function and this

* Fix for collision calculation

* Revert "Fix for collision calculation"

This reverts commit 697d4747c7e3187f98b8407b7c365230ef618781.

yashikabhargava committed:
fix grammatical issues in a_first_splash (#20524)

collin-klippel committed:
fix: Remove extra “the” (#20519)

JaimeObregon committed:
Fixes a mistaken word (#20515)

artisticfox8 committed:
Fixing a typo in a code example (#20514)

When I ran this example, I got  Uncaught ReferenceError: element is not defined
  let output = document.getElementById("output");
if (document.elementsFromPoint) {
  let elements = document.elementsFromPoint(30, 20);
  elements.forEach((elt, i) => {
    output.textContent += element.localName;  //element should be elt
    if (i < elements.length - 1) {
      output.textContent += " < ";
    }
  });
} else {
  output.innerHTML = "<span style=\"color: red;\">" +
     "Browser does not support <code>document.elementsFromPoint()</code>" +
     "</span>";
}

rubiesonthesky committed:
fix for code highlight issues (#20527)

Ahmed-Hakeem committed:
Mention dynamic property accessors (#20531)

* improve property accessor page 

As bracket notation can accept any expression according to specs, then we need to mention that in this page.
We also need to make a convention of naming the syntax all over the docs to be consistent, will it be object.propertyName or object.property?, will it be object[property] or object[propertName] or object["propertyName"], as I've seen different conventions all over the docs and I think sticking with one will be better and consistent.

* Fix page

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

OnkarRuikar committed:
fix(Workflow): auto fix cron job not creating a PR (#20536)

OnkarRuikar committed:
Fix JS code errors (#20539)

* Fix JS code errors

* Update files/en-us/web/api/document_object_model/examples/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: run prettier on mdn and mozilla (#20541)

* style: run prettier on mdn and mozilla

* Update files/en-us/mozilla/add-ons/webextensions/api/sidebaraction/seticon/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/mozilla/add-ons/webextensions/api/sidebaraction/seticon/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/mozilla/firefox/releases/30/index.md

Co-authored-by: Nick Schonning <nschonni@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: run prettier on accessibility (#20542)

nschonni committed:
style: run prettier on web/api/e-l (#20544)

* style: run prettier on web/api/e-l

* Update files/en-us/web/api/idledetector/index.md

* Update files/en-us/web/api/idledetector/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: run prettier on web/api/a-d (#20543)

* style: run prettier on web/api/a-d

* Update files/en-us/web/api/animation/starttime/index.md

* Update files/en-us/web/api/animationplaybackevent/currenttime/index.md

* Update files/en-us/web/api/domimplementation/createhtmldocument/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

OnkarRuikar committed:
CSS: fix style value assignments (#20540)

* CSS: fix style value assignments

Style values are string and require unit.

* use single quotes

* Update files/en-us/web/api/analysernode/getfloatfrequencydata/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

sideshowbarker committed:
Make note in Web_Audio_API/Advanced_techniques more prominent (#20553)

Fixes https://github.com/mdn/content/issues/20529

nschonni committed:
style: run prettier on web/api/x and css (#20547)

* style: run prettier on web/api/x and css

* fix: re-escape HTML in alt

* Update files/en-us/web/css/css_animations/using_css_animations/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix: macro wrapping

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

ramiy committed:
`atan()` - New CSS functional notation (#20492)

yin1999 committed:
fix the max-age description (#20555)

Commit:
Adds comma inside image (#20556)

* adds comma

* compressed

Josh-Cena committed:
More explicitly specify template literal syntax (#20532)

Josh-Cena committed:
Rewrite with statement reference (#20533)

AnilSeervi committed:
Replace NaN with Math in global object compare example (#20557)

* (fix) replace NaN with Infinity in global object compare example

* Update files/en-us/web/javascript/reference/global_objects/globalthis/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

de-oz committed:
Fix capitalization (#20560)

* Fix capitalization

* Update files/en-us/web/api/keyboardevent/code/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

de-oz committed:
Remove extra spaces (#20561)

YangFong committed:
Restructure Array.p.values() page (#19991)

* Adjust the bold range

* Undo changes

* Update index.md

* Format

* Update files/en-us/web/javascript/reference/global_objects/array/values/index.md

* Update index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

nschonni committed:
style: run prettier on learn (#20538)

* style: run prettier on learn

* fix: re-escape tags in alt for Markdownlint

* Update files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/common_questions/what_are_browser_developer_tools/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/learn/common_questions/what_are_browser_developer_tools/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix: replace HTML only alt text

* Remove extra trailing spaces

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

jonahallibone committed:
Don't mutate reduce accumulator (#20440)

* Don't modify the function argument

Return a new array in doubling example instead of mutating the `previousValue`

* Updated counting instances example

* Update groupBy example

* Update duplicate removal example

* Updated .filter().map() example

* Add a description

* Fix

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

zjffun committed:
fix the ref of Date (#20566)

OnkarRuikar committed:
fix(workflow): auto fix cron job run link (#20551)

nschonni committed:
style: run prettier on web/api/s-w (#20546)

* style: run prettier on web/api/s-w

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix: markdownlint issues from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
style: run prettier on web/api/m-r (#20545)

* style: run prettier on web/api/m-r

* fix: add BR and rerun prettier for table

* fix: add BR and rerun prettier for table

Josh-Cena committed:
Recommend against EPSILON for equality tests, align related pages (#20493)

* Recommend against EPSILON for equality tests, align related pages

* Update files/en-us/web/javascript/reference/global_objects/number/epsilon/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/epsilon/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/epsilon/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/isnan/index.md

* Update files/en-us/web/javascript/reference/global_objects/number/isnan/index.md

* Update files/en-us/web/javascript/reference/global_objects/number/issafeinteger/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/epsilon/index.md

* Update files/en-us/web/javascript/reference/global_objects/number/epsilon/index.md

* Update files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/max_safe_integer/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

* Update files/en-us/web/javascript/reference/global_objects/number/min_safe_integer/index.md

* Add section on binary encoding

* Some fixes

* Significand -> mantissa

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>
Co-authored-by: Estelle Weyl <estelle@weyl.org>

nschonni committed:
fix: remove Glossary/Bootstrap (#20520)

mfuji09 committed:
The parameter should be provied as a string rather than `String` wrapper object. (#20574)

nschonni committed:
fix: MD029 ordered style (#20579)

cw118 committed:
Run only on origin and update PR author (#20578)

OnkarRuikar committed:
Tag syntax code blocks with js-nolint, part 1 (#19177)

mfuji09 committed:
Fix a link flaw (#20576)

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

OnkarRuikar committed:
JS: avoid creating implicit global variable (#20575)

teoli2003 committed:
Fix typo introduced in #19177 (#20580)

OnkarRuikar committed:
Fix a typo in label tag (#20573)

* Fix a typo in 'lable for'

* Change to 'dwarfs' from 'drawfs'

* Update email list id

* Update files/en-us/web/html/attributes/multiple/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

nschonni committed:
Redirect Boot2Gecko to Firefox_OS (#20571)

estelle committed:
Improve external link (#20567)

Josh-Cena committed:
Align for...in syntax and page structure with for...of (#20559)

* Align for...in syntax and page structure with for...of

* Add a section in for page

nschonni committed:
Remove Glossary/jQuery (#20569)

hamishwillee committed:
FF105 RelNote: TextEncoderStream, TextDecoderStream (#20583)

* FF105 RelNote: TextEncoderStream, TextDecoderStream

* Minor clarification

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

wbamberg committed:
Fixes to drag operations page (#20403)

* Fixes to drag operations page

* Fix a typo

* Fix page title

* Update files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

sideshowbarker committed:
Drop BCD table from @scroll-timeline article (#19087)

* Drop BCD table from @scroll-timeline article

This is a follow-up to https://github.com/mdn/content/pull/19056

https://github.com/w3c/csswg-drafts/commit/fc6766d dropped the
scroll-timeline at-rule from the “Scroll-linked Animations” spec

The feature was only ever implemented in Firefox, and only behind a flag,
and given that it’s been dropped from the spec, it seems very unlikely
to ever be shipped (un-flagged) in Firefox (or in any other browser).

* Update files/en-us/web/css/@scroll-timeline/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/css/@scroll-timeline/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

OnkarRuikar committed:
HTML: remove deprecated <style type='text/css'> attribute (#20577)

* HTML: remove deprecated <style type='text/css'> attribute

* revert the change in release doc

nschonni committed:
style: run prettier on web/g-j (#20548)

* style: run prettier on web/g-j

* fix: re-escape HTML in alt

* Update files/en-us/web/javascript/reference/global_objects/asyncgeneratorfunction/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/guide/houdini/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/guide/mobile/mobile-friendliness/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/html/attributes/rel/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/html/element/marquee/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/html/element/marquee/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/html/element/thead/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* Update files/en-us/web/javascript/reference/global_objects/number/number/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md

* Apply suggestions from code review

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>

* fix: markdownlint issue

* Update files/en-us/web/javascript/reference/global_objects/intl/collator/collator/index.md

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

rubiesonthesky committed:
Update getOutputTimestamp example from webaudio-examples repo (#20516)

dependabot[bot] committed:
chore(deps): bump @mdn/yari from 1.18.36 to 1.18.39 (#20593)

Rumyra committed:
Update ff-project-issue.md

nschonni committed:
style: run prettier on project files (#20522)

de-oz committed:
Fix punctuation (#20594)

LeoMcA committed:
chore(CODEOWNERS): change default owner to @mdn/core-yari-content (#20587)

Reduces review load for @mdn/core-dev.

de-oz committed:
Remove extra spaces (#20595)

rubiesonthesky committed:
HTML code style: Remove trailing slash section, add note about format and Prettier (#20528)

lopezac committed:
Fix spaces code example of Express Tutorial Part 3 (#20598)

Add a space in comments, change from ej:
"//Bob now exists" to "// Bob now exists".
I think it improves reading having a space and it's a convention.

lopezac committed:
Update dot concatenation of Express Tutorial Part 3 (#20599)

I think it improves reading as is more easy view what happens, if the dot is before the word.
From line 451 to 453 is used the convention that I'm trying to add here.
example change from "
  Athlete.
  find().
  where('sport').equals('Tennis')
"
to "
  Athlete
  .find()
  .where('sport').equals('Tennis')
".

estelle committed:
Revert "Fixed broken link" (#20605)

* Revert "Fixed broken link (#16757)"

This reverts commit ed1393d56cd3f87985ed7c491b1bd444775aa86a.

* Update files/en-us/web/http/headers/index.md

estelle committed:
New: CL tutu

estelle committed:
merge issues
